docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -119,3 +197,41 @@ The important part is having a WordPress site installed, and know where and how
 Alternative editors include [Sublime Text](https://www.sublimetext.com/) that is also available across platforms, though is a commercial product; or other free alternatives include [Vim](https://www.vim.org/), [Atom](https://atom.io/), and [Notepad++](https://notepad-plus-plus.org/) all support standard JavaScript style development.
 
 You can use any editor you're comfortable with, it is more a personal preference. The development setup for WordPress block editor is a common JavaScript environment and most editors have plugins and suppport. The key is having a way to open, edit, and save text files.
+
+## Uninstall - Start Over
+
+Here are a few instructions if you need to start over, or want to remove what was installed.
+
+### Local Environment
+
+* You can remove the local environment data for a specific project:
+
+```
+wp-env destroy
+```
+
```

Comment:
How does this work? Do one need to define which specific project to remove?
It would be helpful with additional instructions here.

This is what I am getting:
```
paaljoachimromdahl@Paals-MacBook-Pro ~ % wp-env destroy
✖ No .wp-env.json file found at '/Users/paaljoachimromdahl/.wp-env.json' and could not determine if '/Users/paaljoachimromdahl' is a WordPress installation, a plugin, or a theme.
paaljoachimromdahl@Paals-MacBook-Pro ~ % 
```

I did this:
Went to the gutenberg directory/folder in the root folder for the user.
Added wp-env destroy.
It gave me a message that it could not connect to Docker. (Because I did not have it running.)
I turned on Docker and ran wp-env destroy again.
It removed a WordPress environment. (That is then probably the javascript environment and has nothing to do with the gutenberg folder I have inside the root user folder.)
```
paaljoachimromdahl@Paals-MacBook-Pro ~ % cd gutenberg
paaljoachimromdahl@Paals-MacBook-Pro gutenberg % wp-env destroy
ℹ WARNING! This will remove Docker containers, volumes, and networks associated with the WordPress instance.
? Are you sure you want to continue? Yes
✖ Error while running docker-compose command.
Couldn't connect to Docker daemon. You might need to start Docker for Mac.
paaljoachimromdahl@Paals-MacBook-Pro gutenberg % wp-env destroy
ℹ WARNING! This will remove Docker containers, volumes, and networks associated with the WordPress instance.
? Are you sure you want to continue? Yes
✔ Removed WordPress environment. (in 6s 927ms)
paaljoachimromdahl@Paals-MacBook-Pro gutenberg % 
```

Bottom line is: 
To remove a specific project one needs to go to the directory such as an example `cd gutenberg` enters the gutenberg directory inside the root user folder (or no?). From there remove the WordPress environment.

Reply:
Did you clarify where the Gutenberg folder came from? The wp-env script does not create a gutenberg folder.

Reply:
It looks I have probably forgotten to remove the Gutenberg folder from an earlier install.

I will uninstall and start over.